### ISSUE TITLE
Reproducibility issue

### ISSUE THREAD

**Issue by whikwon** [mateuszbuda/brain-segmentation-pytorch#12]
Hello, thank you for sharing nice work. 
I'm sorry but I can't get the dice score you got. 
Is it able to reproduce the evaluation result without changing any code?

**Comment by mateuszbuda** [comment]
Hi, thanks for trying my code.
If you run training from docker, then validation split is the same and you should get roughly the same (or very similar) dice score.

**Comment by whikwon** [comment]
I've tried using docker but model doesn't predict CS6667(2nd in validation set) well(dice score = 0.0).
I checked the data but it's OK.

**Comment by mateuszbuda** [comment]
Did you train your model from scratch or are you running inference only?

**Comment by whikwon** [comment]
I'm training model from scratch now.

**Comment by whikwon** [comment]
I've checked the inference only result and it was OK.
